packager/babelRegisterOnly.js

```diff
@@ -10,20 +10,20 @@
 
 require('babel-polyfill');
 
-var fs = require('fs');
-var path = require('path');
-
 var _only = [];
 
-function readBabelRC() {
-  var rcpath = path.join(__dirname, 'react-packager', 'rn-babelrc.json');
-  var source = fs.readFileSync(rcpath).toString();
-  return JSON.parse(source);
-}
-
 module.exports = function(onlyList) {
   _only = _only.concat(onlyList);
-  var config = readBabelRC();
-  config.only = _only;
-  require('babel-register')(config);
+
+  require('babel-register')({
+    presets: ['es2015-node4'],
+    plugins: [
+      'syntax-trailing-function-commas',
+      'syntax-object-rest-spread',
```

Comment:
Yes, most babel plugins are split like this, there is the syntax plugin so babel can parse the code using the feature and then the transform plugin that does the transform.